### DISCUSSION TITLE
Is it possible to include a link in an abbreviation description?

### DISCUSSION THREAD

**Discussion by Stephen-Gates** [squidfunk/mkdocs-material#3287] [Q&A] [answered]
I've read about abbreviations and got them working but when I place a markdown or html link in the abbreviation description they render as plain text. 

Is it possible to include a link in an abbreviation description?

**Comment by squidfunk** [comment, marked as answer]
Currently not, because abbreviations are normally just rendered as tooltips by the operating system. However, I'm already working on a solution that will allow for adding arbitrary HTML to abbreviations, similar to code annotations 😊

**Reply by squidfunk** [reply]
Here's the first step to a solution for the problem of building site-wide glossaries. This feature is going to be called "Instant Previews" and allows for rendering sections on other pages inside a tooltip. It doesn't solve the glossary problem entirely, since you still need to link the terms, but it's the first step towards something that I imagine to be far superior to the current abbreviation solution.

https://user-images.githubusercontent.com/932156/144631031-9dbc1d0f-1be4-403f-b502-414bc97712c8.mp4

**Reply by Stephen-Gates** [reply]
This looks really promising. Interested to see how you solve for different use cases, e.g.

- I want to give you a preview of the content at the other end of the link (like in the video above)
- I want to spell out an acronym term (SaaS = Software as a Service)
- I want to spell out and define an acronym (SaaS = Software as a Service = Software as a Service is a software licensing and delivery model in which software is licensed on a subscription basis and is centrally hosted. SaaS is also known as "on-demand software" and Web-based/Web-hosted software. [Read more on Wikipedia](https://en.wikipedia.org/wiki/Software_as_a_service)) 
- I want the option to have a single file that I can include across the site of acronyms, their terms and definitions
- I want the solution to work on mobile 

Will definately being using this feature as soon as it drops 😍

**Reply by squidfunk** [reply]
Thanks for elaborating on the different use cases, very valuable!
